src/scripts/main.js

```diff
@@ -1,5 +1,37 @@
 'use strict';
 
+const spider = document.querySelector('.spider');
+
+const spiderHeight = spider.clientHeight;
+const spiderWidth = spider.clientWidth;
+
 document.addEventListener('click', e => {
-  // write code here
+  const wall = e.target.closest('.wall');
```

Comment:
https://prnt.sc/u5swv2

Actually, I'm not sure if this is supposed to be corrected or not, but when clicking on the area outside the wall there is an error in console.  I think the reason is hidden in line 9.

Reply:
Fixed